apps/OboeTester/app/src/main/cpp/cpu/AudioWorkloadTest.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef AUDIO_WORKLOAD_TEST_H
+#define AUDIO_WORKLOAD_TEST_H
+
+#include <oboe/Oboe.h>
+#include <vector>
+#include <atomic>
+#include <chrono>
+#include <thread>
+#include <cmath>
+#include <algorithm>
+#include <iostream>
+#include <unistd.h> // For CPU affinity
+#include "SynthWorkload.h"
+
+class AudioWorkloadTest : oboe::AudioStreamDataCallback {
+public:
+    struct CallbackStatus {
+        int32_t numVoices;
+        int64_t beginTimeNs;
+        int64_t finishTimeNs;
+        int32_t xRunCount;
+        int32_t cpuIndex;
+    };
+
+    AudioWorkloadTest();
+    int32_t open();
```

Comment:
Add simple method docs and an example of calling the class.
That may increase adoption.